File: festival-web/src/main/java/cn/bdqfork/web/util/VertxUtils.java
package cn.bdqfork.web.util;


import io.vertx.reactivex.core.Vertx;

/**
 * @author bdq
 * @since 2020/1/21
 */
public class VertxUtils {
    private static final Vertx vertx;

    static {
        vertx = Vertx.vertx();
    }

    public static Vertx getVertx() {
        return vertx;
    }
}

File: festival-kotlin-web/src/main/kotlin/cn/bdqfork/kotlin/web/util/VertxUtils.kt
package cn.bdqfork.web.util

import io.vertx.core.Vertx

/**
 * @author bdq
 * @since 2020/1/21
 */
object VertxUtils {
    @JvmStatic
    var vertx: Vertx = Vertx.vertx()

}